src/templates/speciality.js

```diff
@@ -122,6 +122,12 @@ export const pageQuery = graphql`
               url
             }
           }
+          additionalPoster {
+            title
+            file {
+              url
+            }
+          }
```

Comment:
Can you have a go at updating this and the rest of the poster images to use the contentful fluid queries? Then where we use it we'll need to use common `Image` component